src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/DesigntimeLicenseContextSerializer.cs

```diff
@@ -50,8 +50,6 @@ public static void Serialize(Stream o, string cryptoKey, DesigntimeLicenseContex
             }
         }
 
-        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
```

Comment:
@eerhardt : You also made some infra changes so mscorelib.dll would exist after trimming in https://github.com/dotnet/runtime/pull/48527. I don't really understand the changes, so do we keep them or do you want me to remove those changes?

Reply:
Remove them please. We can add them back if/when we need them.